src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private int size;
+
+    public MyHashMap() {
+        this(DEFAULT_CAPACITY);
+    }
+
+    public MyHashMap(int initialCapacity) {
+        buckets = new LinkedList[initialCapacity];
+        size = 0;
+    }
```

Comment:
```suggestion
    public MyHashMap() {
        buckets = new LinkedList[DEFAULT_CAPACITY];
    }
```
0 - default  value for int